addons/account/models/account_move.py

```diff
@@ -3498,6 +3498,10 @@ def _post(self, soft=True):
                 customer_count[invoice.partner_id] += 1
             elif invoice.is_purchase_document():
                 supplier_count[invoice.partner_id] += 1
+            elif invoice.move_type == 'entry':
```

Comment:
Is there a particular reason why you check that the `move_type` is `'entry'`, rather than just an `else`?